pkg/kubelet/kubelet.go

```diff
@@ -1523,16 +1523,15 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Fetch the pull secrets for the pod
-	pullSecrets, err := kl.getPullSecretsForPod(pod)
-	if err != nil {
-		glog.Errorf("Unable to get pull secrets for pod %q: %v", format.Pod(pod), err)
-		return err
+	pullSecrets := kl.getPullSecretsForPod(pod)
+	if len(pullSecrets) == 0 {
+		glog.Warningf("Get no pull secrets for pod %q", format.Pod(pod))
```

Comment:
getPullSecretsForPod() will never return error. and if len(pullSecrets) == 0,just print warning info not err